src/components/ICA/ICA.stories.js

```diff
@@ -30,11 +29,54 @@ const storyProps = ({ value = icaValue, total } = {}) => ({
 
 storiesOf(components('ICA'), module)
   .addDecorator(withA11y)
-  .addDecorator(centered)
-  .add('Default', () => <ICA {...storyProps()} />)
-  .add('Null value', () => <ICA {...storyProps({ value: null })} />)
-  .add('1000 value', () => <ICA {...storyProps({ value: icaValue * 100 })} />)
-  .add('1000000 value', () => (
+  .add('default', () => <ICA {...storyProps()} />)
+  .add('with null value', () => <ICA {...storyProps({ value: null })} />)
+  .add('with 1000 value', () => (
+    <ICA {...storyProps({ value: icaValue * 100 })} />
+  ))
+  .add('with 1000000 value', () => (
     <ICA {...storyProps({ value: icaValue * 100000 })} />
   ))
-  .add('Total', () => <ICA {...storyProps({ total })} />);
+  .add('with total', () => <ICA {...storyProps({ total })} />)
+  .add(
+    'in an ICA wall',
+    () => (
+      <div className="bx--grid bx--grid--full-width">
+        <div className="bx--row">
```

Comment:
This looks good to me, although it's a bit jarring to see this layout: 
![image](https://user-images.githubusercontent.com/5973294/64634074-115e0000-d3c2-11e9-9834-51e6ce632fc7.png)

I first thought that it was the grid wrapping in weird ways and that all of the ICAs were meant to go in one row. A way around this could be to just add a label on each row to differentiate between the two different layouts. Something like this, but perhaps better worded and styled:

![image](https://user-images.githubusercontent.com/5973294/64634410-c2fd3100-d3c2-11e9-82bf-1fc54b340ec2.png)

Reply:
Or the wall preview could just be a single row?